.circleci/config.yml

```diff
@@ -51,6 +51,31 @@ jobs:
             - bin/audius-arm
             - bin/audius-x86
 
+      - run:
+          name: Add binaries to PATH
+          command: |
+            echo 'export PATH=~/dot-slash-audius/bin:$PATH' >> $BASH_ENV
+            source $BASH_ENV
+      - run:
+          name: standup and test creator
+          command: |
+            cd ~/dot-slash-audius
+            mkdir ~/.audius
+            cp ~/dot-slash-audius/sample.audius.conf ~/.audius/creator.conf
+            audius-x86 -c ~/.audius/creator.conf -network=stage -t=$COMPOSE_COMMIT_SHA
+      - run:
+          name: check creator node health
+          command: |
+            timeout=0
+            while [ "$(curl -sko /dev/null -w "%{http_code}" https://localhost/health_check)" = 502 ] && [ "$timeout" -lt 60 ]; do
+              sleep 3
+              ((timeout=timeout+3))
+              echo "Waiting for server to start... $timeout"
+            done
+            cresult="$(curl -Lkv https://localhost/health_check | jq '.data.healthy')"
+            [ "$cresult" = true ] || [ "$cresult" = false ]
+
+
```

Comment:
Any reason why this is copied into the build-job?

Reply:
Sorry, missed your earlier comment on this block. I will consolidate the thread there.

Reply:
yes I brought them all together because I couldn't access the `COMPOSE_COMMIT_SHA ` from another job, wanted your input on if there's a better way to do that?

Reply:
We could either make the test job depend on the build job, or we could delete the test job and just run it in the build job